### PR TITLE
Upgrade Ejabberd to V20.01

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6070

### CODE REVIEW

**Comment by stephdl** [comment]
issue done, as per the topic I propose an upgrade to ejabberd 20.01

**Comment by gsanchietti** [comment]
Ok for the upgrade, but why adding MQTT support which depends on MariaDB?
I sponsor the upgrade without such feature.

**Comment by stephdl** [comment]
- [ ] remember translation

File: root/etc/httpd/conf.d/ejabberd.conf [hunk 1]
```diff
@@ -9,6 +9,6 @@ SSLProxyCheckPeerCN off
 SSLProxyCheckPeerName off
 SSLProxyCheckPeerExpire off
 
-ProxyPass /http-bind https://127.0.0.1:5280/http-bind
-ProxyPassReverse /http-bind https://127.0.0.1:5280/http-bind
+ProxyPass /bosh https://127.0.0.1:5280/bosh
+ProxyPassReverse /bosh https://127.0.0.1:5280/bosh
```
Comment (gsanchietti) [inline root/etc/httpd/conf.d/ejabberd.conf]:
```suggestion
ProxyPass /http-bind https://127.0.0.1:5280/bosh
ProxyPassReverse /http-bind https://127.0.0.1:5280/bosh
```

Comment (stephdl) [reply root/etc/httpd/conf.d/ejabberd.conf]: I did two proxypass : bosh and http-bind

**Comment by Neustradamus** [comment]
You have forgotten all my changes...
- https://github.com/NethServer/nethserver-ejabberd/commits/ejabberd-19
- ...

Can you add a function for 5223 port?
It is not needed by default.
But you can enable by default for old machine.
Admin must to quickly disable in interface.

Note: https://github.com/NethServer/nethserver-ejabberd/search?q=http-bind&unscoped_q=http-bind

**Comment by stephdl** [comment]
Please never comment on merged PR or closed issue, it doesn't make sense

**Pull request merged**